linera-views/README.md

```diff
@@ -11,6 +11,144 @@ atomically persisted in storage).
 
 See the [CONTRIBUTING](../CONTRIBUTING.md) file for how to help out.
 
+## Design
+
+# Key Value Store Clients
```

Comment:
`###`

Reply:
I choose to use `#` in coherence with the `README.md`.